From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Antheas Kapenekakis <git@antheas.dev>
Date: Mon, 13 Jul 2026 20:03:45 +0200
Subject: drop soft reboot breaking autologin

yeah, bypass, then again once your drive is unlocked it is unlocked and it makes the feature useless

Interim solution even so
---
 src/daemon/DaemonApp.cpp | 22 +---------------------
 1 file changed, 1 insertion(+), 21 deletions(-)

diff --git a/src/daemon/DaemonApp.cpp b/src/daemon/DaemonApp.cpp
index f2171ad30601..d8a99efbb611 100644
--- a/src/daemon/DaemonApp.cpp
+++ b/src/daemon/DaemonApp.cpp
@@ -92,27 +92,7 @@ bool DaemonApp::tryLockFirstLogin()
     }
     m_firstloginLock = true;
 
-    QDBusMessage msg = QDBusMessage::createMethodCall(QStringLiteral("org.freedesktop.systemd1"),
-                                                      QStringLiteral("/org/freedesktop/systemd1"),
-                                                      QStringLiteral("org.freedesktop.DBus.Properties"),
-                                                      QStringLiteral("Get"));
-
-    msg << QStringLiteral("org.freedesktop.systemd1.Manager") << QStringLiteral("SoftRebootsCount");
-
-    const QDBusMessage reply = QDBusConnection::systemBus().call(msg);
-
-    if (reply.type() == QDBusMessage::ErrorMessage) {
-        qWarning() << "DBus error:" << reply.errorName() << "-" << reply.errorMessage();
-        return true;
-    }
-
-    const QVariant soft_reboot_count = qvariant_cast<QDBusVariant>(reply.arguments().at(0)).variant();
-    if (!soft_reboot_count.isValid()) {
-        qWarning() << "DBus variant is invalid:" << reply;
-        return true;
-    }
-
-    return soft_reboot_count.toUInt() == 0;
+    return true;
 };
 }
 
